src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -716,6 +720,22 @@ Namespace Microsoft.CodeAnalysis.Semantics
             Return New LazyPropertyReferenceExpression([property], instance, [property], argumentsInEvaluationOrder, _semanticModel, syntax, type, constantValue, isImplicit)
         End Function
 
+        Private Function CreateBoundWithLValueExpressionPlaceholder(boundWithLValueExpressionPlaceholder As BoundWithLValueExpressionPlaceholder) As IInstanceReferenceExpression
+            Dim syntax As SyntaxNode = boundWithLValueExpressionPlaceholder.Syntax
+            Dim type As ITypeSymbol = boundWithLValueExpressionPlaceholder.Type
+            Dim constantValue As [Optional](Of Object) = ConvertToOptional(boundWithLValueExpressionPlaceholder.ConstantValueOpt)
+            Dim isImplicit As Boolean = boundWithLValueExpressionPlaceholder.WasCompilerGenerated
+            Return New InstanceReferenceExpression(_semanticModel, syntax, type, constantValue, isImplicit)
+        End Function
+
+        Private Function CreateBoundWithRValueExpressionPlaceholder(boundWithRValueExpressionPlaceholder As BoundWithRValueExpressionPlaceholder) As IInstanceReferenceExpression
```

Comment:
>CreateBoundWithRValueExpressionPlaceholder [](start = 25, length = 42)

It look like there is no test coverage for this new code, it appears BoundWithRValueExpressionPlaceholder is only used with ```With``` statement.

Reply:
There are a few: https://github.com/dotnet/roslyn/pull/22076/files/9d3257299252cbc345d2ff1e718a1312249dce4e#diff-f63cce822741d92606b3910d4c4d5e14L56 as an example. I also added https://github.com/dotnet/roslyn/pull/22076/files/9d3257299252cbc345d2ff1e718a1312249dce4e#diff-a9ede0042063c66b3a49be3a727f7260R42 for this scenario.

Reply:
> There are a few

Please confirm at least one of the the ```With``` statement scenarios is actually using BoundWithRValueExpressionPlaceholder node, it is hard to tell whether this is the case.

Reply:
I confirmed it debugging though the code manually.